kubernetes_asyncio/stream/ws_client_test.py

```diff
@@ -61,6 +61,7 @@ def __aiter__(self):
         with patch('kubernetes_asyncio.client.api_client.rest', mock):
 
             api_client = WsApiClient()
+            api_client.configuration.host = 'https://localhost'
```

Comment:
because the openapi-generator uses `http://localhost` as a default hostname and the swagger-codegen used `https://localhost`.